src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSign1Message.cs

```diff
@@ -0,0 +1,283 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Formats.Cbor;
+using System.Runtime.Versioning;
+
+namespace System.Security.Cryptography.Cose
+{
+    public sealed class CoseSign1Message : CoseMessage
+    {
+        private const string SigStructureCoxtextSign1 = "Signature1";
+        private const int Sign1ArrayLegth = 4;
+
+        internal CoseSign1Message(CoseHeaderMap protectedHeader, CoseHeaderMap unprotectedHeader, byte[]? content, byte[] signature, byte[] protectedHeaderAsBstr)
+            : base(protectedHeader, unprotectedHeader, content, signature, protectedHeaderAsBstr) { }
+
+        [UnsupportedOSPlatform("browser")]
```

Comment:
Presumably you added this because the analyzer said to... but that seems odd... browser-compat is supposed to be opt-in, so I wouldn't expect this to trip.

Reply:
Yes, the errors sprout from `ECDsa.SignData()`, `RSA.SignData()`, and `RSASignaturePadding.Pss` but I don't see any unsupported attributes in those APIs. So I don't really know what's causing them.

Reply:
>  browser-compat is supposed to be opt-in, so I wouldn't expect this to trip.

That is right, by defualt it should not warn for cross platform builds, but browser team wanted to see the warning for cross-platform builds in runtime repo